.eslintignore

```diff
@@ -0,0 +1,3 @@
+# These components use ES7 class statics.
+src/components/Layout/Layout.js
```

Comment:
https://stackoverflow.com/questions/38296761/how-to-support-es7-in-eslint

Reply:
*Stage 3 class statics https://github.com/tc39/proposal-static-class-features